graphconstruction/creatingGraph.py

```diff
@@ -173,8 +175,10 @@ def prepare_dataset(rfilename, num_event, num_start=0):
         dist_eta = distance.cdist(eta, eta, 'cityblock')
 
         dist = np.sqrt(dist_phi ** 2 + dist_eta ** 2)
-        edge_source = np.where((dist < 0.4) & (dist != 0))[0]
-        edge_target = np.where((dist < 0.4) & (dist != 0))[1]
+        deltaRSetting = 0.8
```

Comment:
move this to the very beginning and add one comment here, to make it big and visible?